packages/core/package.json

```diff
@@ -36,8 +36,10 @@
     "vscode-ws-jsonrpc": "^0.0.2-1",
     "ws": "^3.0.0",
     "yargs": "^9.0.1",
-    "react": "^16.2.0",
-    "react-dom": "^16.2.0"
+    "react": "^16.4.1",
```

Comment:
Since you have bumped up the version, I would have expected a `yarn.lock` change too.

[minor]: Also, can you move up the `@types/react` and `@types/react-dom` to the other `@types/x`?